packages/astro/src/core/ssr/css.ts

```diff
@@ -4,7 +4,7 @@ import path from 'path';
 import { viteifyURL } from '../util.js';
 
 // https://vitejs.dev/guide/features.html#css-pre-processors
-export const STYLE_EXTENSIONS = new Set(['.css', '.pcss', '.scss', '.sass', '.styl', '.stylus', '.less']);
+export const STYLE_EXTENSIONS = new Set(['.css', '.pcss', '.postcss', '.scss', '.sass', '.styl', '.stylus', '.less']);
```

Comment:
👍🏻 yup this matches [Vite‘s supported styles](https://github.com/vitejs/vite/blob/main/packages/vite/src/node/plugins/css.ts#L85).